lib/explorer/backend/series.ex

```diff
@@ -344,6 +344,8 @@ defmodule Explorer.Backend.Series do
   # Struct
   @callback field(s, String.t()) :: s
 
+  @callback index_of(s, valid_types()) :: integer()
```

Comment:
I just realised this should have a return type of integer() | nil not just integer().  I can make a new pr for it if you think it's necessary.

Reply:
No that's ok, I'll just change it. Thanks for letting us know!